core/templates/components/forms/slider/audio-slider.component.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Wrapper over mat-slider for audio-bar.
+ */
+
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+@Component({
+  selector: 'oppia-audio-slider',
+  template: `<mat-slider
+    color="accent"
+    [value]="value"
+    style="width: 100%;"
+    [thumbLabel]="thumbLabel"
+    [max]="max"
+    (change)="setProgress($event)"
+    tick-interval="auto"
+    [step]="1"
+    aria-label="audio-slider">
+  </mat-slider>`,
```

Comment:
This is being removed in the migration of the exploration editor (which is going on at the moment). I decided to not spend more time on this file. Ideally, this would be in an HTML file of its own but since it is ephemeral, we can let it be.

Reply:
> This is being removed in the migration of the exploration editor (which is going on at the moment)

Do you mean we are going to remove the `audio-slider.component.ts`?